src/PowerShellEditorServices.Hosting/BuildInfo.cs

```diff
@@ -0,0 +1,10 @@
+namespace Microsoft.PowerShell.EditorServices.Hosting
+{
+    public static class BuildInfo
+    {
+        public static readonly string BuildVersion = "<development-build>";
+        public static readonly string BuildOrigin = "<development>";
+        public static readonly System.DateTime? BuildTime = System.DateTime.Parse("2019-12-06T21:43:41");
+        public static readonly bool IsPreview = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'IsPreview', the compiler will do that for you.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4579423)